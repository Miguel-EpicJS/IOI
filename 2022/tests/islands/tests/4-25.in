4e5150f5a11b21d9ac3b44f2259e7a3452343ad4
100 2000
0 49
0 49
87 17
87 17
38 72
38 72
35 53
35 53
76 83
76 83
47 69
47 69
80 55
80 55
80 67
80 67
82 85
82 85
77 17
77 17
5 8
5 8
39 29
39 29
12 20
12 20
77 37
77 37
39 54
39 54
62 27
62 27
38 65
38 65
16 3
16 3
24 69
24 69
9 0
9 0
79 67
79 67
90 70
90 70
82 68
82 68
48 69
48 69
39 42
39 42
7 58
7 58
54 19
54 19
62 95
62 95
4 40
4 40
12 45
12 45
75 51
75 51
43 97
43 97
20 23
20 23
21 83
21 83
12 72
12 72
85 37
85 37
81 44
81 44
21 5
21 5
0 68
0 68
2 51
2 51
21 18
21 18
64 19
64 19
21 95
21 95
87 55
87 55
62 15
62 15
76 32
76 32
87 57
87 57
90 24
90 24
87 25
87 25
62 22
62 22
97 49
97 49
9 31
9 31
35 78
35 78
38 61
38 61
25 72
25 72
27 32
27 32
61 37
61 37
39 17
39 17
11 29
11 29
87 59
87 59
50 85
50 85
35 17
35 17
28 54
28 54
84 99
84 99
88 91
88 91
30 51
30 51
82 67
82 67
16 19
16 19
3 95
3 95
30 33
30 33
46 69
46 69
58 69
58 69
4 54
4 54
47 0
47 0
2 70
2 70
82 87
82 87
99 57
99 57
20 1
20 1
16 88
16 88
35 42
35 42
21 41
21 41
38 33
38 33
4 2
4 2
24 69
24 69
52 18
52 18
21 78
21 78
76 40
76 40
87 55
87 55
92 23
92 23
35 91
35 91
64 57
64 57
94 88
94 88
77 33
77 33
34 73
34 73
15 18
15 18
9 18
9 18
82 33
82 33
35 40
35 40
93 63
93 63
76 40
76 40
4 90
4 90
36 69
36 69
24 70
24 70
87 94
87 94
4 99
4 99
73 67
73 67
26 23
26 23
59 67
59 67
38 51
38 51
40 44
40 44
96 69
96 69
76 57
76 57
5 64
5 64
35 27
35 27
98 0
98 0
7 79
7 79
2 51
2 51
7 70
7 70
87 37
87 37
74 73
74 73
20 97
20 97
38 27
38 27
95 88
95 88
39 65
39 65
87 27
87 27
22 78
22 78
16 64
16 64
8 64
8 64
21 49
21 49
11 88
11 88
62 73
62 73
60 95
60 95
19 44
19 44
4 15
4 15
21 71
21 71
0 91
0 91
24 70
24 70
71 70
71 70
82 64
82 64
5 65
5 65
7 0
7 0
21 4
21 4
76 79
76 79
76 53
76 53
37 80
37 80
25 65
25 65
87 27
87 27
77 57
77 57
94 88
94 88
70 17
70 17
16 73
16 73
19 18
19 18
82 23
82 23
60 43
60 43
45 51
45 51
30 45
30 45
87 89
87 89
29 25
29 25
35 56
35 56
61 57
61 57
36 23
36 23
21 13
21 13
69 61
69 61
76 19
76 19
50 69
50 69
10 86
10 86
1 14
1 14
26 0
26 0
16 62
16 62
39 40
39 40
81 17
81 17
24 70
24 70
13 19
13 19
36 51
36 51
38 63
38 63
62 6
62 6
76 65
76 65
76 80
76 80
38 54
38 54
34 80
34 80
4 88
4 88
70 88
70 88
91 59
91 59
74 18
74 18
52 69
52 69
95 64
95 64
75 34
75 34
5 53
5 53
9 45
9 45
94 33
94 33
4 5
4 5
92 78
92 78
58 33
58 33
87 8
87 8
39 72
39 72
16 54
16 54
68 63
68 63
72 40
72 40
2 63
2 63
38 40
38 40
20 85
20 85
21 64
21 64
9 67
9 67
66 51
66 51
30 54
30 54
61 44
61 44
90 27
90 27
42 17
42 17
3 63
3 63
0 79
0 79
16 18
16 18
96 49
96 49
6 55
6 55
49 19
49 19
35 91
35 91
79 44
79 44
95 13
95 13
3 0
3 0
0 99
0 99
34 57
34 57
4 91
4 91
75 67
75 67
48 25
48 25
4 83
4 83
48 51
48 51
51 31
51 31
2 22
2 22
93 69
93 69
4 35
4 35
15 56
15 56
35 33
35 33
20 55
20 55
47 95
47 95
88 99
88 99
28 69
28 69
4 55
4 55
38 18
38 18
16 53
16 53
10 19
10 19
83 42
83 42
38 15
38 15
62 78
62 78
60 6
60 6
28 23
28 23
24 7
24 7
43 97
43 97
76 63
76 63
5 77
5 77
81 60
81 60
12 51
12 51
82 72
82 72
72 27
72 27
75 64
75 64
58 70
58 70
90 49
90 49
62 78
62 78
16 54
16 54
71 23
71 23
9 49
9 49
39 76
39 76
24 55
24 55
21 10
21 10
90 34
90 34
38 16
38 16
4 13
4 13
9 70
9 70
82 8
82 8
62 32
62 32
39 72
39 72
87 13
87 13
35 64
35 64
23 19
23 19
62 37
62 37
7 85
7 85
62 60
62 60
40 64
40 64
4 15
4 15
92 59
92 59
29 18
29 18
82 99
82 99
62 70
62 70
35 15
35 15
38 95
38 95
62 40
62 40
62 65
62 65
86 41
86 41
87 61
87 61
68 80
68 80
16 64
16 64
77 34
77 34
16 8
16 8
15 61
15 61
76 55
76 55
91 32
91 32
44 32
44 32
38 79
38 79
49 37
49 37
3 78
3 78
5 68
5 68
25 37
25 37
74 0
74 0
94 28
94 28
23 41
23 41
7 51
7 51
39 33
39 33
87 12
87 12
39 68
39 68
75 69
75 69
90 33
90 33
38 60
38 60
35 58
35 58
34 72
34 72
87 63
87 63
21 11
21 11
5 13
5 13
4 19
4 19
50 42
50 42
66 51
66 51
62 25
62 25
39 56
39 56
7 51
7 51
66 85
66 85
50 70
50 70
85 91
85 91
40 18
40 18
73 67
73 67
67 18
67 18
97 56
97 56
4 30
4 30
16 68
16 68
86 0
86 0
4 65
4 65
3 84
3 84
7 85
7 85
44 18
44 18
82 32
82 32
26 95
26 95
39 12
39 12
48 67
48 67
39 80
39 80
28 33
28 33
76 63
76 63
70 97
70 97
8 59
8 59
86 32
86 32
86 13
86 13
35 55
35 55
20 51
20 51
27 40
27 40
87 83
87 83
48 12
48 12
21 57
21 57
8 13
8 13
52 51
52 51
50 97
50 97
4 84
4 84
75 44
75 44
84 13
84 13
47 8
47 8
4 63
4 63
21 72
21 72
75 69
75 69
21 63
21 63
92 37
92 37
71 83
71 83
56 32
56 32
39 32
39 32
61 80
61 80
26 69
26 69
4 65
4 65
76 2
76 2
62 83
62 83
35 43
35 43
58 0
58 0
16 11
16 11
62 37
62 37
35 15
35 15
84 45
84 45
21 72
21 72
68 55
68 55
94 5
94 5
39 33
39 33
67 18
67 18
91 63
91 63
39 96
39 96
75 95
75 95
81 42
81 42
66 59
66 59
77 37
77 37
14 67
14 67
4 21
4 21
98 70
98 70
76 17
76 17
35 79
35 79
89 51
89 51
31 44
31 44
86 97
86 97
26 57
26 57
25 64
25 64
94 51
94 51
76 56
76 56
71 56
71 56
22 85
22 85
21 83
21 83
62 5
62 5
26 89
26 89
35 79
35 79
32 53
32 53
48 73
48 73
36 37
36 37
63 78
63 78
29 70
29 70
39 29
39 29
96 0
96 0
47 6
47 6
25 37
25 37
80 32
80 32
24 69
24 69
0 19
0 19
9 69
9 69
5 65
5 65
58 91
58 91
82 59
82 59
2 80
2 80
58 54
58 54
82 78
82 78
76 9
76 9
58 72
58 72
35 88
35 88
14 51
14 51
91 80
91 80
16 99
16 99
50 11
50 11
70 6
70 6
16 1
16 1
76 5
76 5
16 40
16 40
0 78
0 78
74 34
74 34
10 95
10 95
63 53
63 53
39 46
39 46
76 13
76 13
66 10
66 10
85 19
85 19
76 38
76 38
16 15
16 15
48 8
48 8
82 61
82 61
9 83
9 83
11 0
11 0
39 79
39 79
16 28
16 28
96 95
96 95
24 68
24 68
4 43
4 43
62 18
62 18
48 23
48 23
92 6
92 6
76 27
76 27
20 49
20 49
43 97
43 97
71 59
71 59
33 67
33 67
38 34
38 34
88 32
88 32
43 66
43 66
77 68
77 68
87 25
87 25
21 8
21 8
82 88
82 88
82 61
82 61
76 49
76 49
50 37
50 37
46 77
46 77
2 83
2 83
62 77
62 77
7 54
7 54
22 51
22 51
76 13
76 13
38 81
38 81
12 70
12 70
16 34
16 34
74 51
74 51
38 70
38 70
58 97
58 97
82 10
82 10
73 17
73 17
76 55
76 55
66 69
66 69
44 57
44 57
88 72
88 72
35 17
35 17
76 6
76 6
62 33
62 33
87 54
87 54
43 95
43 95
82 44
82 44
15 44
15 44
39 13
39 13
1 37
1 37
62 27
62 27
66 18
66 18
45 72
45 72
52 91
52 91
94 59
94 59
76 54
76 54
26 95
26 95
38 77
38 77
4 90
4 90
40 64
40 64
12 23
12 23
25 57
25 57
11 15
11 15
31 70
31 70
62 15
62 15
76 49
76 49
16 49
16 49
26 0
26 0
82 72
82 72
4 78
4 78
16 13
16 13
39 18
39 18
62 14
62 14
64 42
64 42
59 17
59 17
14 95
14 95
0 23
0 23
90 23
90 23
28 85
28 85
4 83
4 83
62 63
62 63
64 33
64 33
62 67
62 67
62 83
62 83
62 15
62 15
62 72
62 72
66 19
66 19
25 17
25 17
4 27
4 27
21 33
21 33
46 18
46 18
95 85
95 85
9 41
9 41
21 54
21 54
82 56
82 56
39 78
39 78
21 40
21 40
25 78
25 78
25 40
25 40
73 33
73 33
81 56
81 56
26 0
26 0
35 6
35 6
29 46
29 46
48 15
48 15
4 56
4 56
16 77
16 77
92 41
92 41
35 91
35 91
40 57
40 57
62 65
62 65
82 86
82 86
12 34
12 34
76 33
76 33
82 57
82 57
7 34
7 34
35 64
35 64
16 56
16 56
39 80
39 80
29 23
29 23
4 15
4 15
76 72
76 72
38 40
38 40
48 85
48 85
8 17
8 17
4 73
4 73
54 61
54 61
39 68
39 68
63 44
63 44
90 41
90 41
62 90
62 90
82 67
82 67
7 65
7 65
76 8
76 8
62 67
62 67
78 19
78 19
52 59
52 59
35 65
35 65
29 78
29 78
76 15
76 15
1 27
1 27
52 97
52 97
28 56
28 56
96 48
96 48
48 41
48 41
16 9
16 9
43 23
43 23
40 99
40 99
60 95
60 95
24 27
24 27
76 63
76 63
21 80
21 80
87 83
87 83
62 44
62 44
16 78
16 78
49 57
49 57
16 34
16 34
65 99
65 99
39 73
39 73
4 99
4 99
87 99
87 99
4 89
4 89
28 85
28 85
76 33
76 33
62 19
62 19
16 57
16 57
81 15
81 15
78 32
78 32
68 40
68 40
28 95
28 95
38 25
38 25
54 80
54 80
3 51
3 51
16 80
16 80
11 65
11 65
38 14
38 14
62 65
62 65
62 72
62 72
6 63
6 63
62 77
62 77
89 41
89 41
82 61
82 61
88 49
88 49
49 57
49 57
77 73
77 73
16 77
16 77
73 19
73 19
27 49
27 49
1 69
1 69
82 41
82 41
35 73
35 73
5 37
5 37
26 27
26 27
15 40
15 40
68 8
68 8
93 69
93 69
25 61
25 61
64 19
64 19
75 23
75 23
4 83
4 83
21 74
21 74
31 34
31 34
69 49
69 49
52 63
52 63
28 61
28 61
76 15
76 15
39 18
39 18
46 53
46 53
58 97
58 97
87 65
87 65
60 78
60 78
34 57
34 57
87 32
87 32
92 97
92 97
4 55
4 55
5 32
5 32
21 54
21 54
71 85
71 85
20 55
20 55
82 99
82 99
84 36
84 36
98 85
98 85
87 74
87 74
76 54
76 54
88 59
88 59
16 49
16 49
46 92
46 92
3 0
3 0
54 79
54 79
21 59
21 59
87 34
87 34
52 74
52 74
31 81
31 81
40 79
40 79
79 37
79 37
86 6
86 6
66 53
66 53
26 70
26 70
76 39
76 39
61 33
61 33
4 37
4 37
43 88
43 88
76 40
76 40
2 8
2 8
88 79
88 79
35 58
35 58
95 37
95 37
21 89
21 89
76 34
76 34
58 95
58 95
70 25
70 25
86 96
86 96
30 51
30 51
74 77
74 77
90 59
90 59
4 68
4 68
76 25
76 25
98 99
98 99
38 13
38 13
52 23
52 23
16 15
16 15
35 42
35 42
46 15
46 15
58 47
58 47
16 36
16 36
38 77
38 77
77 40
77 40
4 8
4 8
69 44
69 44
35 15
35 15
89 70
89 70
21 44
21 44
71 30
71 30
30 26
30 26
21 37
21 37
77 32
77 32
10 18
10 18
39 25
39 25
62 4
62 4
83 78
83 78
98 95
98 95
6 18
6 18
87 0
87 0
96 0
96 0
38 18
38 18
76 72
76 72
4 41
4 41
77 34
77 34
50 97
50 97
93 32
93 32
21 19
21 19
49 59
49 59
9 0
9 0
30 70
30 70
76 40
76 40
39 53
39 53
39 61
39 61
76 78
76 78
76 80
76 80
62 82
62 82
16 68
16 68
83 44
83 44
76 77
76 77
62 6
62 6
36 13
36 13
82 28
82 28
19 44
19 44
4 54
4 54
61 67
61 67
47 83
47 83
32 57
32 57
2 57
2 57
15 55
15 55
7 68
7 68
71 61
71 61
74 3
74 3
50 69
50 69
93 67
93 67
38 32
38 32
62 72
62 72
65 8
65 8
4 70
4 70
4 77
4 77
50 51
50 51
76 78
76 78
88 78
88 78
58 95
58 95
43 5
43 5
19 53
19 53
21 32
21 32
19 44
19 44
75 98
75 98
82 77
82 77
39 23
39 23
47 52
47 52
98 69
98 69
24 59
24 59
92 40
92 40
38 77
38 77
66 70
66 70
31 54
31 54
47 67
47 67
10 70
10 70
4 13
4 13
38 37
38 37
46 33
46 33
31 97
31 97
62 13
62 13
39 11
39 11
31 85
31 85
73 13
73 13
25 68
25 68
2 56
2 56
28 72
28 72
40 59
40 59
78 18
78 18
96 65
96 65
4 65
4 65
66 95
66 95
38 88
38 88
4 56
4 56
76 59
76 59
92 69
92 69
61 44
61 44
3 70
3 70
4 54
4 54
50 51
50 51
39 42
39 42
90 70
90 70
35 76
35 76
39 68
39 68
39 61
39 61
36 64
36 64
21 17
21 17
37 59
37 59
21 72
21 72
94 63
94 63
82 6
82 6
16 25
16 25
62 19
62 19
39 55
39 55
93 97
93 97
21 49
21 49
22 75
22 75
36 37
36 37
89 2
89 2
93 0
93 0
66 70
66 70
49 42
49 42
98 95
98 95
97 88
97 88
62 44
62 44
27 18
27 18
31 15
31 15
16 59
16 59
10 15
10 15
36 18
36 18
45 40
45 40
16 65
16 65
38 5
38 5
82 57
82 57
85 69
85 69
43 85
43 85
62 63
62 63
56 67
56 67
42 53
42 53
69 73
69 73
36 50
36 50
21 40
21 40
38 80
38 80
21 50
21 50
4 88
4 88
21 32
21 32
38 55
38 55
7 85
7 85
76 33
76 33
89 51
89 51
87 16
87 16
77 54
77 54
12 25
12 25
97 45
97 45
83 18
83 18
38 40
38 40
41 95
41 95
83 59
83 59
1 45
1 45
34 65
34 65
93 99
93 99
38 87
38 87
46 97
46 97
68 33
68 33
23 63
23 63
3 97
3 97
39 72
39 72
76 65
76 65
75 85
75 85
4 25
4 25
69 70
69 70
82 6
82 6
98 93
98 93
39 54
39 54
13 19
13 19
7 42
7 42
4 53
4 53
2 79
2 79
9 69
9 69
87 27
87 27
43 23
43 23
21 18
21 18
24 33
24 33
16 73
16 73
39 6
39 6
48 13
48 13
35 23
35 23
62 61
62 61
26 83
26 83
87 8
87 8
4 80
4 80
39 54
39 54
30 69
30 69
41 18
41 18
92 55
92 55
84 97
84 97
16 33
16 33
62 95
62 95
60 44
60 44
73 80
73 80
21 80
21 80
92 94
92 94
21 99
21 99
58 55
58 55
25 73
25 73
14 51
14 51
47 95
47 95
70 18
70 18
76 91
76 91
77 17
77 17
76 18
76 18
4 63
4 63
62 72
62 72
37 59
37 59
35 68
35 68
36 40
36 40
74 97
74 97
84 70
84 70
10 97
10 97
35 99
35 99
89 18
89 18
82 17
82 17
36 0
36 0
76 34
76 34
62 27
62 27
39 73
39 73
16 64
16 64
66 51
66 51
73 19
73 19
82 33
82 33
87 64
87 64
82 5
82 5
62 42
62 42
91 56
91 56
26 44
26 44
21 91
21 91
98 85
98 85
60 45
60 45
87 19
87 19
50 40
50 40
48 85
48 85
35 67
35 67
72 8
72 8
11 51
11 51
39 40
39 40
1 83
1 83
4 83
4 83
16 68
16 68
47 95
47 95
14 90
14 90
83 64
83 64
35 34
35 34
76 65
76 65
20 69
20 69
68 64
68 64
24 68
24 68
38 71
38 71
5 91
5 91
1 95
1 95
38 79
38 79
96 32
96 32
77 56
77 56
20 67
20 67
51 33
51 33
76 73
76 73
35 44
35 44
86 17
86 17
83 57
83 57
92 45
92 45
22 88
22 88
62 42
62 42
93 9
93 9
70 99
70 99
84 25
84 25
7 69
7 69
28 71
28 71
58 83
58 83
38 46
38 46
62 67
62 67
87 57
87 57
76 55
76 55
38 29
38 29
