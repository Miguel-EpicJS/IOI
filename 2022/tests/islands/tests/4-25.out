91041670623a69783795ed82bda33426101ce311
OK
1
34
1054 604 1748 1084 1704 1776 684 1742 308 450 1028 1776 1704 1084 1748 604 1054 1055 604 1748 1084 1704 1776 1028 450 308 1742 684 1776 1704 1084 1748 604 1055
